openrtb2/deal.go

```diff
@@ -66,6 +68,39 @@ type Deal struct {
 	//   bid on this deal. Omission implies no advertiser restrictions.
 	WADomain []string `json:"wadomain,omitempty"`
 
+	// Attribute:
+	//   guar
+	// Type:
+	//   integer, default 0
+	// Description:
+	//   Indicates that the deal is of type guaranteed and the bidder must
+	//   bid on the deal, where 0 = not a guaranteed deal, 1 = guaranteed deal.
+	Guar int8 `json:"guar,omitempty"`
+
+	// Attribute:
+	//   mincpmpersec
+	// Type:
+	//   float
+	// Definition:
+	//   Minimum CPM per second. This is a price floor for video or audio
+	//   impression opportunities, relative to the duration of bids an
+	//   advertiser may submit.
+	MinCPMPerSec float64 `json:"mincpmpersec,omitempty"`
+
+	// Attribute:
+	//   durfloors
+	// Type:
+	//   object array
+	// Description:
+	//   An array of DurFloors objects (Section 3.2.35) indicating the floor
+	//   prices for video creatives of various durations that the buyer may bid with.
+	DurFloors []DurFloors `json:"durfloors,omitempty"`
```

Comment:
Description in spec: 
```
Container for floor price by duration information, to be used if a given deal is eligible for video or audio demand. An array of DurFloors objects (see Section 3.2.35).
```

Reply:
Good find. This was a copy/paste error. Fixed.